man/ostree-prepare-root.xml

```diff
@@ -113,6 +113,14 @@ License along with this library. If not, see <https://www.gnu.org/licenses/>.
                 <term><varname>sysroot.readonly</varname></term>
                 <listitem><para>A boolean value; the default is false.  If this is set to <literal>true</literal>, then the <literal>/sysroot</literal> mount point is mounted read-only.</para></listitem>
           </varlistentry>
+            <varlistentry>
+                <term><varname>composefs.enable</varname></term>
+                <listitem><para>This can be yes, no or maybe, the default is no.  If this is set to <literal>yes</literal>, then composefs is always used, and the boot fails if it is not available. If set to <literal>maybe</literal>, then composefs is used if supported.</para></listitem>
+          </varlistentry>
+            <varlistentry>
+                <term><varname>composefs.keyfile</varname></term>
```

Comment:
(None of the below is blocking)

I'm fine with this, however...we could also introduce `/etc/ostree/initramfs-root-binding.key` or so and if it exists we automatically honor it  It's a bit easier to use.

If we did this it does probably argue to also support `composefs.enable=signed` again to require signature validation.

But I do also see this "initramfs root binding" as something necessary for composefs, but also logically distinct.  We could also do this even if composefs isn't enabled - basically if the key is present, use it for verification.

A final thing - it looks like the main ostree-sign ed25519 code supports basic "keyrings" that are newline separated files.

It could be beneficial to do so here too and reuse the same code.